okokyst_metadata.py

```diff
@@ -1,6 +1,127 @@
 # The addStationMetadata function needs to be modified if new areas are to be
 # plotted using this package
-def addStationMeadata(station, CTDConfig, options = None):
+
+
+serveys_lookup_table = {
```

Comment:
Rename `serveys_lookup_table`to `surveys_loopkup_table`

Reply:
why surveys_loopkup_table? It sounds like typo

Reply:
Well `surveys` is the correct word and `serveys` is a typo

Reply:
aah, ok. I meant `loopkup` from your first comment was a typo, but this is another one, ok, I'll change

Reply:
Ah now I see what you meant - our two typos cancel eachother out